README.md

```diff
@@ -0,0 +1,19 @@
+# Hello Microverse
+
+> This is my first Microverse project.
+
+## Built With
+
+- html
+- css
+- github
+
+## Author
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/dadadon)
+- Twitter: [@twitterhandle](https://twitter.com/davianberoni)
```

Comment:
Please change the generic handles to your name and last name and/or your actual handle for that platform. It'll give you more visibility and make your documentation look more professional.